packages/rs-drive-abci/src/execution/engine/finalize_block_proposal/v0/mod.rs

```diff
@@ -247,6 +239,8 @@ where
         }
         .into();
 
+        self.update_drive_cache(&block_execution_context, platform_version)?;
```

Comment:
We consume BlockExecutionContext to call update_state_cache. If you look at the update_state_cache function you will see that we aren't using drive cache there and not supposed to since update_state_cache should deal with only state cache.

Reply:
Okay I reviewed the order change and we *should* be okay because of the height guard for queries.